staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/types.go

```diff
@@ -74,12 +74,12 @@ type CustomResourceDefinitionSpec struct {
 // CustomResourceConversion describes how to convert different versions of a CR.
 type CustomResourceConversion struct {
 	// strategy specifies how custom resources are converted between versions. Allowed values are:
-	// - `None`: The converter only change the apiVersion and would not touch any other field in the custom resource.
-	// - `Webhook`: API Server will call to an external webhook to do the conversion. Additional information
+	// - `"None"`: The converter only change the apiVersion and would not touch any other field in the custom resource.
+	// - `"Webhook"`: API Server will call to an external webhook to do the conversion. Additional information
```

Comment:
hm. It's really not a good sign that the only time this fired, it was a collision between enum value and field name.

Reply:
but I agree with your fix here.

Reply:
@sftim do you know if there is a specific way API docs should mention enum values?

Reply:
I believe we don't have any conventions for comments that get turned into API metadata. I might be wrong though.

I would like there to be conventions but don't think I have the capacity to help manage that effort.